dockerfiles/che/Dockerfile.ubi8

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+ARG CHE_DASHBOARD_VERSION=next
+ARG CHE_WORKSPACE_LOADER_VERSION=next
+
+FROM quay.io/eclipse/che-dashboard:${CHE_DASHBOARD_VERSION} as che_dashboard_base
+FROM quay.io/eclipse/che-workspace-loader:${CHE_WORKSPACE_LOADER_VERSION} as che_workspace_loader_base
```

Comment:
is this used in the Che build / release? If not, we might consider having a rhel.Dockerfile (to support building in Brew) instead of a Dockerfile.ubi8 (to show how to build with ubi8 but then not actually release bits this way?).

Reply:
ok. I'll remove ubi8 image this time. Let's do rhel on next step